web/views.py

```diff
@@ -40,7 +42,29 @@ def project_detail(request,project_id):
 	project = get_object_or_404(Project, project_id=project_id)
 	return render(request, 'pages/project_detail.html', {'project': project})
 
+def project_new(request):
+	if request.method == "POST":
+		projectform = ProjectForm (request.POST)
+		if projectform.is_valid():
+			project = projectform.save()
+			return redirect('web.views.project_detail', project_id=project.project_id)
+	else:
+		projectform = ProjectForm()
+
+	return render(request, 'pages/project_edit.html', {'projectform': projectform})
```

Comment:
I was following only the Djangogirls tutorials solution, where they have two methods